core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -135,3 +124,24 @@
     width: 220px;
   }
 </style>
+
+<link rel="stylesheet" href="/third_party/static/perfectscrollbar-0.6.16/css/perfect-scrollbar.css" />
```

Comment:
Ah, sorry. I meant: "Drop " /" at end. Should be `<link...`."

(I.e., you interpreted it right!)